doc/BEGINNERS_TUTORIAL.org

```diff
@@ -0,0 +1,189 @@
+Spacemacs maximized.
+** Quitting (restarting)
+Save the changes you've made to the dotfile with SPC f s and then quit emacs by
+SPC q q. You can return to this tutorial by clicking it on the home screen!
+* Additional features, tips and troubleshooting
+** Org mode
+Org mode is one of the best features of Spacemacs and enough reason to warrant
+its use. Org mode's official description tells that it is "for
+keeping notes, maintaining todo lists, planning projects, and authoring
+documents with a fast and effective plain-text system", but this gives only a
+small inkling of its versatility. If you do any kind of writing at all, chances
+are that Org mode will make it easier and more fun. This tutorial was written in
+Org mode.
+
+Install the Org layer and open this tutorial. Press shift-TAB repeatedly and observe that
+this cycles the visibility of the contents of different headlines. Press t in
+normal mode and observe that you can add TODO tags on headlines. Press M-k
+(Alt-k, M stands for "modifier") or M-j in normal mode and see how you can
+quickly move parts of the document around.
+
+This is not even scratching the surface of Org mode, so you should look into
+https://github.com/syl20bnr/spacemacs/tree/master/layers/org for more
+information. Googling for Org mode tutorials is also very helpful in finding out
+the most useful features of it!
+** Version control - the intelligent way
+Version control means keeping track of the changes and edits you have made to
+your document. Often version control is done by saving different versions of the
+document with different names, such as "document version 13" and so on. This is
+crude in many ways: if you want to, for example, re-add something you deleted, you have to
+manually open several past versions of the document to find the one with the
+deleted part, and then copy-paste it to the most recent file. More complicated
+edits will be harder still. Fortunately, there is a much better way.
+Git is the most popular version control system for programmers, but it can be as
+useful for people that are writing school or scientific papers, fiction or blog
+posts as well.
+
+Install the git layer, restart Spacemacs and open a file you want to version
+control. You can check the status of your file by pressing SPC g s. Select the
+folder your file is in. You will be prompted whether you want to create a
+repository in the folder. Select yes. You will see a list of "Untracked files":
+navigate to the file you want to track and press s to "stage changes". You might
+be prompted to save the file: save it if necessary. Now the new file needs to be
+commited: press c and c again. Two windows pop up: one showing the changes
+you've made since the last edit (in this case, the whole document) and another
+prompting for a commit message. Write "Initial commit", press ESC to exit back
+to normal mode and press :wq to write and quit the commit message.
```

Comment:
instead of `:wq` most people use `SPC m c` to confirm and `SPC m a` to abort or `C-c C-c` and `C-c C-k`.